src/scooch.js

```diff
@@ -127,33 +142,43 @@
 
     var Scooch = (function($, Utils) {
         var defaults = {
-                dragRadius: 10
-              , moveRadius: 20
-              , animate: true
-              , autoHideArrows: false
-              , rightToLeft: false
-              , classPrefix: 'm-'
-              , classNames: {
-                    outer: 'scooch'
-                  , inner: 'scooch-inner'
-                  , item: 'item'
-                  , center: 'center'
-                  , touch: 'has-touch'
-                  , dragging: 'dragging'
-                  , active: 'active'
-                  , inactive: 'inactive'
-                  , fluid: 'fluid'
-                }
+            dragRadius: 10
+            , moveRadius: 20
+            , animate: true
+            , autoHideArrows: false
+            , rightToLeft: false
+            , infinite: false
+            , classPrefix: 'm-'
+            , classNames: {
+                outer: 'scooch'
+                , inner: 'scooch-inner'
+                , item: 'item'
+                , center: 'center'
+                , touch: 'has-touch'
+                , dragging: 'dragging'
+                , active: 'active'
+                , inactive: 'inactive'
+                , fluid: 'fluid'
```

Comment:
@jansepar I refactored the source code to use this `var` declaration style. In this block however we are just declaring the one `defaults` object, not separate `var`s on each line.